### ISSUE TITLE
Niantic servers with problem?

### ISSUE THREAD

**Issue by vinsilsim** [AHAAAAAAA/PokemonGo-Map#3297]
Hello everybody, I am conducting research in central park and look at the return.

2016-08-03 10:17:29,787 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,788 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,789 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,790 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,792 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,793 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,795 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,795 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,800 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,802 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,804 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,805 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:17:29,806 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms

**Comment by hagehg** [comment]
As of today, you will only get valid returns for scans 10 seconds apart. Unless the scans you pasted are all from different accounts that should be the reason for your problem.

**Comment by vinsilsim** [comment]
this is by ip or account?

**Comment by MrKetchumAll** [comment]
Change your -sd # to -sd 10 and see how that goes.

**Comment by vinsilsim** [comment]
root@instance-3:/home/PokemonGo-Map-develop# python runserver.py -a google -u ********** -p ********* -l "Central Park, NY" -st 10 -sd 10 -H 10.128.0.4 -P 5060
2016-08-03 10:27:12,642 [    MainThread][     runserver] [   INFO] Parsed location is: 40.7711/-73.9742/0.0000 (lat/lng/alt)
2016-08-03 10:27:12,648 [    MainThread][        models] [   INFO] Connecting to MySQL database on *********.
2016-08-03 10:27:12,674 [ search_thread][        search] [   INFO] Search loop 0 starting
2016-08-03 10:27:12,674 [ search_thread][        search] [   INFO] Attempting login to Pokemon Go.
2016-08-03 10:27:12,674 [ search_thread][   auth_google] [   INFO] Google login for: *********
2016-08-03 10:27:13,128 [ search_thread][   auth_google] [   INFO] Google Login successful.
2016-08-03 10:27:13,347 [ search_thread][        search] [   INFO] Login to Pokemon Go successful.
2016-08-03 10:27:13,393 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:27:23,434 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:27:33,480 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:27:43,555 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:27:53,595 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:28:03,638 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:28:13,796 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:28:23,860 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:28:33,960 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:28:44,156 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:28:54,322 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:29:04,370 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:29:14,453 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:29:24,503 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:29:34,546 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:29:44,637 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:29:54,713 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:30:04,762 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:30:14,910 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:30:24,954 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:30:35,054 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:30:45,143 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:30:55,263 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:31:05,309 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:31:15,348 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:31:25,389 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:31:35,428 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:31:45,467 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 10:31:55,521 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms

**Comment by tarthim** [comment]
As of ~10 minutes ago I can't login to PTC accounts anymore through PokemonGo-Map. Getting a "Failed to login to Pokemon Go. Trying again in 5 seconds" error. Running -sd 5, st 5.

http://ispokemongodownornot.com/ is showing nothing. Something else is up?
Might have something to do with this as well?

**Comment by vinsilsim** [comment]
@tarthim everything seems ok to login just do not find pokemon

**Comment by MrKetchumAll** [comment]
You didn't happen to filter all the Mons did you vin?

**Comment by vinsilsim** [comment]
@MrKetchumAll No, that's how it was when normal. I went to sleep with everything working and now finds nothing in the research

**Comment by Shadowgar** [comment]
I now scan using a minimum of 50 accounts. Scanning fine so far.

**Comment by vinsilsim** [comment]
@Shadowgar  i use 200 accounts, this is no problem. I dont think but my ip can get banned.

**Comment by MrKetchumAll** [comment]
How do you guys go about making that many accounts? Gotta make email, then sign up, then go confirm. There a shortcut?

**Comment by vinsilsim** [comment]
@MrKetchumAll https://temp-mail.org dont need create e-mails.

**Comment by MrKetchumAll** [comment]
Ah, that's ingenious vin.

**Comment by MrKetchumAll** [comment]
So what's the password for these temp mail? How do I use it to log into the game?

**Comment by vinsilsim** [comment]
this limitation is -sd10 IP or per user? someone can tell me?

**Comment by vinsilsim** [comment]
@MrKetchumAll temp email dont need password, you just change account name, and refresh box!

**Comment by MrKetchumAll** [comment]
Yeah, I mean when you use it to log into the pokemon game. How do you log into your search without a password to the account?

**Comment by ELusiveLPFAN** [comment]
OMG when you register account you setup password...

**Comment by MrKetchumAll** [comment]
It's -sd 10 for each user/account. No IP involve.

**Comment by hagehg** [comment]
@vinsilsim According to a reddit post (https://www.reddit.com/r/pokemongodev/comments/4vwtia/ip_address_limitation_added/), Niantic may have added IP limitations aswell

**Comment by Bizadon** [comment]
@vinsilsim @Shadowgar Hey guys, can you please answer several questions? 
You are running develop branch, am I right? Can you copy here example of how you start the scan? (dont need your login and passwords, just can write like log1 log2... pass1 pass2 such as example). Sorry for that questions, I just cant find exact information of it how to run with multiple accounts. And one more question, are you running on server or on local machine? At what OS?

**Comment by excitedbox** [comment]
username [user1, user2, user3, etc]
password whateverpass (if they all have same pass or like above if all have diff pass)

put that in the config file

**Comment by Bizadon** [comment]
@excitedbox You use develop or master branch? You mean config.ini file? If I rightly understand, this config needs to write setting there, not in cmd and just run runserver.py, am I right? So if that it would be like: 
# Authentication settings
#auth-service:   ptc       # ptc (default) or google
#username: user1, user2, user3, etc
#password: pass1, pass2, pass3, etc
right?
You run on windows with python and external ip address?

**Comment by realbunny** [comment]
Im running -sd 5, it works for me, getting my pokemons

**Comment by Oricul** [comment]
I'm currently experiencing this. Migrated my servers to new boxes, was working great for ~4 hours, then suddenly, all Pokemon began falling off the results. Now both servers are reporting 0 Pokemon in all locations. I'm not sure what to do at this point... SD is set to 10, but I'm also using the honeycomb batch, which I think may ignore the delay? Any suggestions?
